migration/idempotent/001-base.sql

```diff
@@ -2073,14 +2074,20 @@ DECLARE
     label_array int[];
     next_epoch BIGINT;
     deletion_epoch BIGINT;
+    epoch_duration INTERVAL;
 BEGIN
     next_epoch := present_epoch + 1;
     -- technically we can delete any ID <= current_epoch - 1
     -- but it's always safe to leave them around for a bit longer
     deletion_epoch := present_epoch - 4;
 
+    SELECT _prom_catalog.get_default_value('epoch_duration')::INTERVAL INTO epoch_duration;
+    IF epoch_duration IS NULL THEN
```

Comment:
`_prom_catalog.get_default_value` will not return a null value. The value you put in the `_prom_catalog.initial_default` view is what will be returned if the user has not specified a default (even if the user specifies NULL as the default).

I would remove the IF statement and add change line 2084 to use `INTO STRICT`